man/systemd-analyze.xml

```diff
@@ -770,6 +770,24 @@ Service b@0.service not loaded, b.socket cannot be started.
         operate on files inside the specified image path <replaceable>PATH</replaceable>.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--offline=<replaceable>BOOL</replaceable></option></term>
+
+        <listitem><para>With <command>security</command>, performs an offline security review (i.e. does
+        not have to rely on PID 1 to acquire security information) of the specified unit
+        file(s) and generates a detailed analysis table, assigning "exposure level" values to various
```

Comment:
don#t dupicate the explanation of "security" here. it's already explained below. Just say how this switch alters the behaviour of "security"